<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrasladoActivoFijo</class>
 <widget class="QWidget" name="TrasladoActivoFijo">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>975</width>
    <height>625</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tralado de Bienes al Inventario de Activo Fijo</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="3" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Codigo Bien:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Clase:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Cuenta:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Correlativo:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Especifico:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_10" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLineEdit" name="cuentaNO">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="maxLength">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="claseNO">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="maxLength">
            <number>3</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="correlativoNO">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="maxLength">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="especificoNO">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="maxLength">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="consultaBien">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="12" column="2" colspan="3">
    <widget class="QLineEdit" name="observaciones">
     <property name="maxLength">
      <number>148</number>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="label_18">
     <property name="font">
      <font>
       <family>Impact</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>AF-M2</string>
     </property>
    </widget>
   </item>
   <item row="13" column="4">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="guardarTraslado">
        <property name="text">
         <string>Guardar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="finalizarTransaccion">
        <property name="text">
         <string>Trasladar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelar">
        <property name="text">
         <string>Cancelar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_17">
     <property name="text">
      <string>Observaciones:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="6">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Bien:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="1">
       <widget class="QWidget" name="widget_8" native="true">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="1">
          <widget class="QWidget" name="widget_4" native="true">
           <layout class="QFormLayout" name="formLayout_2">
            <item row="2" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Modelo:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Marca:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="marca">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="modelo">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QWidget" name="widget_5" native="true">
           <layout class="QFormLayout" name="formLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Valor:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Serie:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDoubleSpinBox" name="valorBien">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="locale">
               <locale language="Spanish" country="ElSalvador"/>
              </property>
              <property name="wrapping">
               <bool>false</bool>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="specialValueText">
               <string/>
              </property>
              <property name="keyboardTracking">
               <bool>true</bool>
              </property>
              <property name="maximum">
               <double>10000000.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="serie">
              <property name="maxLength">
               <number>25</number>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QWidget" name="widget_6" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Adquirido:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="fechaAquisicion">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="1" colspan="3">
          <widget class="QWidget" name="widget_7" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Descripcion:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="descripcion">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QWidget" name="widget_6" native="true">
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Cuenta: </string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="cuentaAsignada">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>Especifico: </string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="nombreEspecifico">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" rowspan="4" colspan="5">
    <widget class="QTableWidget" name="tablaBienes">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QToolButton" name="botonAgregarBien">
     <property name="toolTip">
      <string>Agregar Bien</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../ContablesPrincipal.qrc">
       <normaloff>:/iconos/mas.png</normaloff>:/iconos/mas.png</iconset>
     </property>
    </widget>
   </item>
   <item row="5" column="5">
    <widget class="QToolButton" name="botonEliminarBien">
     <property name="toolTip">
      <string>Eliminar Bien</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../ContablesPrincipal.qrc">
       <normaloff>:/iconos/menos.png</normaloff>:/iconos/menos.png</iconset>
     </property>
    </widget>
   </item>
   <item row="6" column="5">
    <widget class="QToolButton" name="nuevoBien">
     <property name="toolTip">
      <string>Nuevo Bien</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../ContablesPrincipal.qrc">
       <normaloff>:/iconos/filenew.png</normaloff>:/iconos/filenew.png</iconset>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QLineEdit" name="totalIngreso">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>Total:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Codigo Traslado:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="codigoTraslado"/>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Tipo Traslado:</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="temporal">
           <property name="text">
            <string>Temporal</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="definitivo">
           <property name="text">
            <string>Definitivo</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QWidget" name="widget_9" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Unidad que Recibe:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="unidadRecibe"/>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Inventario:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="anioInventario"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../ContablesPrincipal.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cuentaNO</sender>
   <signal>textChanged(QString)</signal>
   <receiver>claseNO</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>611</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>claseNO</sender>
   <signal>textChanged(QString)</signal>
   <receiver>correlativoNO</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>566</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>669</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>claseNO</sender>
   <signal>textChanged(QString)</signal>
   <receiver>especificoNO</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>577</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>838</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>correlativoNO</sender>
   <signal>textChanged(QString)</signal>
   <receiver>marca</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>710</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>485</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>correlativoNO</sender>
   <signal>textChanged(QString)</signal>
   <receiver>valorBien</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>710</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>688</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>correlativoNO</sender>
   <signal>textChanged(QString)</signal>
   <receiver>modelo</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>654</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>527</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>correlativoNO</sender>
   <signal>textChanged(QString)</signal>
   <receiver>serie</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>728</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>724</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>correlativoNO</sender>
   <signal>textChanged(QString)</signal>
   <receiver>fechaAquisicion</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>750</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>890</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>correlativoNO</sender>
   <signal>textChanged(QString)</signal>
   <receiver>descripcion</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>745</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>549</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
